test_runner/performance/pgvector/loaddata.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+import numpy as np
+import pandas as pd
+import psycopg2
+from pgvector.psycopg2 import register_vector
+from psycopg2.extras import execute_values
+
+directory_path = (
+    "/Users/neon/work/vanilla/dbpedia-entities-openai3-text-embedding-3-large-1536-1M/data"
+)
+
+
+# Connection to PostgreSQL
+
+with psycopg2.connect(
+    "postgresql://neondb_owner:<secret>@ep-floral-thunder-w1gzhaxi.eu-west-1.aws.neon.build/neondb?sslmode=require"
+) as conn:
+    register_vector(conn)
+    with conn.cursor() as cursor:
+        # List and sort Parquet files
+        parquet_files = [f for f in os.listdir(directory_path) if f.endswith(".parquet")]
```

Comment:
Optional: I prefer using [`pathlib`](https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module) instead of `os`, so if you're up to it, you can try using it

Reply:
done